docs/api/datasets.rst

```diff
@@ -110,6 +116,12 @@ Non-geospatial Datasets
 
 :class:`VisionDataset` is designed for datasets that lack geospatial information. These datasets can still be combined using :class:`ConcatDataset <torch.utils.data.ConcatDataset>`.
 
+.. csv-table:: C = classification,  R = regression, S = semantic segmentation, I = instance segmentation, T = time series, CD = change detection, OD = object detection
```

Comment:
These keys are slightly different than what we used in our paper. If we're going to use CD and OD should we also use SS, II, and TS? Or should we stick to the keys we use in the paper?

Reply:
They are different because we didn't have object detection in the paper and used "D" for change detection.